### PR TITLE
Add Tests for ExecuteCommand

### LINKED ISSUE #297: Add tests for the execute method of the ExecuteCommand
https://github.com/doctrine/migrations/blob/master/lib/Doctrine/DBAL/Migrations/Tools/Console/Command/ExecuteCommand.php

### PR DESCRIPTION
**Pull request by chrisguitarguy**
Done with the Symfony console command tester.

Closes #297

### CODE REVIEW

File: tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Command/ExecuteCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            '--dry-run' => true,
+            '--query-time' => true,
+        ], ['interactive' => false]);
+
+        $this->assertSame(0, $tester->getStatusCode());
+    }
+
+    protected function setUp()
+    {
+        $this->questions = $this->getMock(QuestionHelper::class);
```
Comment (mikeSimonson) [inline tests/Doctrine/DBAL/Migrations/Tests/Tools/Console/Command/ExecuteCommandTest.php]: You just need to add something similar to the test here https://github.com/doctrine/migrations/blob/master/bin/doctrine-migrations.php#L74-L78 and then the last test will pass.

Thanks a lot for the PR.

**Comment by mikeSimonson** [comment]
@chrisguitarguy Thanks a lot for the tests.
There is just one little change to make so that the last test pass.

**Comment by chrisguitarguy** [comment]
Should be good to go @mikeSimonson, didn't think about the `DialogHelper` thing when I originally wrote the tests. Had to make a few other changes to support the Symfony 2.3 `CommandTester`.

**Comment by mikeSimonson** [comment]
@chrisguitarguy Thanks you are awesome

**Comment by chrisguitarguy** [comment]
No problem, @mikeSimonson. On vacation this week, so I thought I'd get back to helping out with migrations a bit.

**Comment by mikeSimonson** [comment]
still awesome

**Pull request merged**